provisioner/terraform/testdata/resources/presets/external-module/main.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     coder = {
       source  = "coder/coder"
-      version = "2.1.3"
+      version = "2.3.0-pre2"
```

Comment:
Now that https://github.com/coder/terraform-provider-coder/pull/373 is merged we should cut a release and reference it here.